modules/services/window-managers/i3-sway/lib/functions.nix

```
@@ -4,9 +4,10 @@ with lib;
 
 rec {
   criteriaStr = criteria:
-    "[${
-      concatStringsSep " " (mapAttrsToList (k: v: ''${k}="${v}"'') criteria)
-    }]";
+    let
+      toCriteria = k: v:
+        if builtins.isBool v && v then "${k}" else ''${k}="${v}"'';
```

Given `{ floating = true; }` this will produce `[floating]`.

Is it possible to negate a criteria?

---

If it's false, maybe just exclude it entirely?

```suggestion
        if builtins.isBool v then (if v then "${k}" else "") else ''${k}="${v}"'';
```

Also should have a test here.